CONTRIBUTING.md

```diff
@@ -101,12 +101,13 @@ number vor the tag.
 3. Run `BUMP=(bugfix|feature|breaking) make bump_version` to update the `pydo`
    version.
     * `BUMP` also accepts `(patch|minor|major)`
-4. Make a pull request with this change. It should be separate from PRs
-   containing chagnes to the library (including regenerated code).
-5. *Once the version bump PR has been pushed*, tag the commit to trigger the
+4. Run `make generate` to update the version in the codebase.
```

Comment:
Shouldn't `make bump_version` update the version? I guess we just didn't follow our own instructions?

Reply:
the `pydo.VERSION` is pulled from [this line](https://github.com/digitalocean/pydo/pull/144/files#diff-16c29cde913e51b86dc104daa8854b03b577aac00b7e0273c22b4cfc7bc6a3d7R7). This is part of the generated code so it would be best to not ask contributors to mess with generated code files. Running `make generate` will update this line. One option is to add the `generate` command to the end of the `bump_version` command to make it **one** command that updates the version in ALL places in the codebase.